packages/addons/packages.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
This file is named `packages.json`, but most likely was meant to be `package.json`.